src/protocol/models.ts

```diff
@@ -58,15 +56,15 @@ export interface ProofData {
 // ZKPRequest is a request for zkp proof
 export interface ZKPRequest {
   id: number;
-  circuit_id: string;
+  circuitId: string;
   optional?: boolean;
   rules: unknown;
 }
 
 // ZKPResponse is a response with a zkp
 export interface ZKPResponse {
   id: number;
-  circuit_id: string;
+  circuitId: string; // `circuitId` compatibility with golang implementation.
```

Comment:
remove comment

Reply:
I believe that inconsistent keys can create the question "Why are the keys different?" this comment is an explanation. Because somebody can change to consistent key `circuit_id`.